src/rest_api/classes/clone/widgets/overview.clj

```diff
@@ -118,22 +119,7 @@
 
 (defn expression-patterns [clone]
   {:data (when-let [ep (first (:expr-pattern/_clone clone))]
-           (let [h (first (:expr-pattern/gene ep))]
-             (let [gene (:expr-pattern.gene/gene h)]
-               {:certainty (generic-functions/certainty h)
-                :expression_pattern {:taxonomy "all"
-                                     :class "expr_pattern"
-                                     :label (str "Expression pattern for "
-                                                 (if-let [n (or (:gene/public-name gene)
-                                                                (:gene/id gene))]
-                                                   n
-                                                   "" ))
-                                     :id (:expr-pattern/id ep)}
-                :reference (when-let [paper (first (:expr-pattern/reference ep))]
-                             (:paper/id (:expr-pattern.reference/paper paper)))
-                :gene (if (empty? gene) nil (pack-obj gene))
```

Comment:
Maybe `(if-not (empty? gene) (pack-obj gene)`

Reply:
I thought it looked odd too. It was old code that I was moving. I am fixing it